ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -204,12 +204,26 @@ Loader {
             return fetchMoreMessagesButtonComponent
         case Constants.messageContentType.systemMessagePrivateGroupType:
             return privateGroupHeaderComponent
+        case Constants.messageContentType.systemMessagePinnedMessage:
+            return systemMessagePinnedMessageComponent
         case Constants.messageContentType.gapType:
             return gapComponent
         case Constants.messageContentType.newMessagesMarker:
             return newMessagesMarkerComponent
-        default:
+        case Constants.messageContentType.messageType:
+        case Constants.messageContentType.stickerType:
+        case Constants.messageContentType.emojiType:
+        case Constants.messageContentType.transactionType:
+        case Constants.messageContentType.imageType:
+        case Constants.messageContentType.audioType:
+        case Constants.messageContentType.communityInviteType:
+        case Constants.messageContentType.discordMessageType:
             return messageComponent
+        case Constants.messageContentType.unknownContentType:
+            // NOTE: We could display smth like "unknown message type, please upgrade Status to see it".
+            return null
```

Comment:
I think you remove the return, as it would use the `default` one instead, but I could be wrong. Anyway, it's not important

Reply:
Yeah I know, I added it explicitly because with this comment it looked quite strange without a ’return’ 😑